staticcheck.conf

```diff
@@ -0,0 +1 @@
+checks = ["all", "-ST1005", "-ST1000", "-ST1016"]
```

Comment:
FYI: This won't be necessary once https://github.com/kudobuilder/kudo/pull/994 is merged because the linter will be configured to skip generated code.

Reply:
awesome! I can wait for #994 to land.

Reply:
#994 has landed 🛬

Reply:
and merged!